scripts/make_UKB_study_table.py

```diff
@@ -50,22 +51,25 @@ def main(input_path: str, output_path: str) -> None:
     manifest.loc[:, "ancestry_replication"] = ""
 
     # Ouput required columns
-    cols = OrderedDict(
-        [
-            ("study_id", "study_id"),
-            ("pmid", "pmid"),
-            ("pub_date", "pub_date"),
-            ("pub_journal", "pub_journal"),
-            ("pub_title", "pub_title"),
-            ("pub_author", "pub_author"),
-            ("ancestry_initial", "ancestry_initial"),
-            ("ancestry_replication", "ancestry_replication"),
-            ("n_initial", "n_initial"),
-            ("n_cases", "n_cases"),
-            ("n_replication", "n_replication"),
-        ]
-    )
-    manifest = manifest.loc[:, list(cols.keys())].rename(columns=cols)
+    cols = [
+        "study_id",
+        "pmid",
+        "pub_date",
+        "pub_journal",
+        "pub_title",
+        "pub_author",
+        "trait_reported",
+        "ancestry_initial",
+        "ancestry_replication",
+        "n_initial",
+        "n_cases",
+        "n_replication",
+    ]
+
+    manifest = manifest.filter(items=cols).drop_duplicates()
+
+    # Assert trait_reported is not empty
+    assert manifest["trait_reported"].notna().all(), "There are studies where the trait is not defined."
```

Comment:
I'm wondering if it would make sense to print out these violating study ids to make the troubleshooting faster.

Reply:
Good idea.
I'd suggest doing it in the script that merges all studies, as here we would only catch the UKB ones. Is that OK?

Reply:
I agree, if you can make this last change I can merge the branch